README.md

```diff
@@ -44,10 +44,10 @@ For example, on most UNIX-like systems, you can setup the hosts file with:
 ./wpt make-hosts-file | sudo tee -a /etc/hosts
 ```
 
-And on Windows (note this requires an Administrator privileged shell):
+And on Windows (note this requires in PowerShell with Administrator privileges):
```

Comment:
How about "this requires PowerShell with Administrator privileges" or "this must be run in PowerShell with Administrative privileges"?

Reply:
I was trying to figure out how MS refer to it; I think something like the latter is probably what we want.

Reply:
Does this now look better?

Reply:
Much!